### PR TITLE
✨ Add image disk format

### PR DESCRIPTION
**Pull request by maelk**
**What this PR does / why we need it**:
In order to use raw image streaming, Ironic nodes need to be configured. There is an option added to BMO : https://github.com/metal3-io/baremetal-operator/pull/533 . This PR adds support for the disk format option in CAPM3

### CODE REVIEW

**Comment by maelk** [comment]
/hold
Needs to be merged in BMO, then change go.mod to remove the replace

File: go.mod [hunk 1]
```diff
@@ -30,3 +30,5 @@ require (
 )
 
 replace k8s.io/client-go => k8s.io/client-go v0.19.0-alpha.2
+
+replace github.com/metal3-io/baremetal-operator => github.com/Nordix/baremetal-operator v0.0.0-20200527073027-54918b79c5e7
```
Comment (kashifest) [inline go.mod:34]: Didn't understand this

Comment (kashifest) [reply go.mod:34]: Aha, ok you are gonna remove this. Ignore the comment

**Comment by kashifest** [comment]
lgtm

**Comment by maelk** [comment]
/hold cancel
merged in BMO, so we can proceed. I'll remove the replace and re-run the tests

**Pull request merged**